completions/git.completion.sh

```diff
@@ -624,7 +624,7 @@ __git_commands () {
 	then
 		printf "%s" "${GIT_TESTING_COMMAND_COMPLETION}"
 	else
-		git help -a|egrep '^  [a-zA-Z0-9]'
+		git help -a|egrep "^  [a-z]*"|cut -d" " -f4
```

Comment:
@avlach Will this fix POSIX-compatible?

Reply:
Yes, this work with bash under posix mode. I have tested with latest Fedora release 30. 

~~~
wasp  ~  bash --posix 
bash-5.0$ bash --version 
GNU bash, version 5.0.7(1)-release (x86_64-redhat-linux-gnu)
Copyright (C) 2019 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
bash-5.0$ git help -a|egrep "^  [a-z]*"|cut -d" " -f4
add
am
archive
bisect
branch
...
stripspace
~~~